static/js/components/GcnTags.jsx

```diff
@@ -190,10 +172,14 @@ const GcnTags = ({ gcnEvent, show_title = false }) => {
               />
             ) : (
               <Chip
-                className={styles[tag]}
                 size="small"
                 label={tag}
                 key={tag}
+                style={{
+                  backgroundColor: gcn_tags_classes
+                    ? gcn_tags_classes[tag]
+                    : "#999999",
```

Comment:
What if gcn_tags_classes exists but the tag isn't in it? It looks like this will still get it from gcn_tags_classes and yield null, using the default MUI color rather than the "#999999" you've set. Same comment for GcnEvents.jsx

Reply:
@Theodlz Good point. I think I fixed it.

Reply:
Yep looks like it works! Feel free to merge whenever you want.